.travis.yml

```diff
@@ -1,9 +1,10 @@
-# before_install:
-#   - gem update --system '2.4.5'
 language: ruby
+cache: bundler
 rvm:
-  - 2.0.0
-  - 2.1.10
-  - 2.2.5
-  - 2.3.3
-  - 2.4.0
+  - 2.0
+  - 2.1
+  - 2.2
+  - 2.3
```

Comment:
Since Ruby 2.4 will reach EOL in march this year I think it's safe enough to skip those versions.
(source: https://www.ruby-lang.org/en/downloads/branches/)

Reply:
Awesome! I thought since this library works on all versions, I leave it as-is, but that will make maintenance easier in the future :)